Dockerfile

```diff
@@ -26,18 +30,87 @@ RUN go get github.com/kevinburke/go-bindata/...
 RUN --mount=type=cache,target=/root/.cache/go-build make bin
 RUN --mount=type=cache,target=/root/.cache/go-build make bin/entrypoint
 
+#--------------------------------------------------------------------
+# imgbase builds the "img" tool and all of its dependencies
+#--------------------------------------------------------------------
+
+# We build a fork of img for now so we can get the `img inspect` CLI
+# Watch this PR: https://github.com/genuinetools/img/pull/324
+FROM hashicorp.jfrog.io/docker/golang:alpine AS imgbuilder
+
+RUN apk add --no-cache \
+	bash \
+	build-base \
+	gcc \
+	git \
+	libseccomp-dev \
+	linux-headers \
+	make
+
+RUN git clone https://github.com/mitchellh/img.git /img
+WORKDIR /img
+RUN git checkout inspect
+RUN go get github.com/go-bindata/go-bindata/go-bindata
```

Comment:
we use `github.com/kevinburke/go-bindata/...` elsewhere

Reply:
Yeah, I'm using this one cause this is the one `img` requires haha. 
This is copied from their Dockerfile.